File: q2.cpp
#include<iostream>
using namespace std;
int main(){
    float c;
    float k;
    float f;
    cout<<"Enter c ";
    cin>>c;
    cout<<"Enter k ";
    cin>>k;
    cout<<"Enter f ";
    cin>>f;
    f=(c*9/5)+32;
    k=c+273.15;
    c=(f-32)*5/9;
    cout<<"c to f "<<f<<endl;
    cout<<"c to k "<<k<<endl;
    cout<<"f to c "<<c<<endl;

}

File: q2.java
import java.util.*;
public class q2 {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number: ");
        int a=sc.nextInt();
        if(a<=0){
            System.out.println("Invalid number");
            return;
        }
        
        for(int i=1;i<=10;i++){
            System.out.println(a+" X "+i+ " = " + a*i);
        }
    }
    
}
